document/Supplementary-Note.md

```diff
@@ -179,16 +179,22 @@ Table 1: Some useful documentation about source code comment and documentation:
 | CodeAsDocumentation                                                                     | http://martinfowler.com/bliki/CodeAsDocumentation.html              |
 | 
 
- 
-
- 
 
-     
-    
- 
- 
- 
+## Git Large File System (LFS)
```

Comment:
@laurent can you add here the section (Section 1) . Please change also the other section names in the manuscripts.